### ISSUE TITLE
Fetching a zone in bind config format

### ISSUE THREAD

**Issue by mikehardenize** [cloudflare/node-cloudflare#39]
You can fetch a cloudflare zone by calling:

`GET zones/$zone_id/dns_records`

This is possible with node-cloudflare. However, you can fetch it in bind configuration syntax if you append /export:

`GET zones/$zone_id/dns_records/export`

This does not appear to be possible with node-cloudflare. Please consider adding this functionality.

**Comment by mikehardenize** [comment]
FWIW, I added the following to DNSRecords.js:

```js
bindConfig: method({
  method: 'GET',
  path: 'export',
  json: false,
}),
```

Allowing me to call:

```js
cf.dnsRecords.bindConfig(zone_id);
```

This worked fine. I haven't submitted it as a PR as I suspect you would maybe want to expand "browse" instead, or give the function a different name?

I'm happy to submit as a PR, with or without changes. Let me know...

**Comment by mikehardenize** [comment]
Additionally, I know this feature isn't in the current set of cloudflare API documentation, but I believe that is an oversight. I have reached out to cloudflare support regarding the issue and they responded:

    Thank you for bringing this to our attention, I believe this should be documented,
    as we do document /import. I have reached out to the team who owns the
    documentation to see if we can get this updated to include /export.

    In the meantime, I have confirmed that it is just a GET request with the endpoint
    being /export, if you do need/want to use the endpoint.

As another data point, they use this API end-point themselves on their own frontend to allow users to export zones in this format.

**Comment by mikehardenize** [comment]
Not heard anything, so submitting as a PR

**Comment by terinjokes** [comment]
In general, I prefer not to add functionality that isn't documented.

**Comment by mikehardenize** [comment]
FWIW, I got another response from Cloudflare this morning, to say:

    We appreciate your patience here. We've heard back that our API documentation should
    be updated to include this information toward the latter half of the month.

    We're happy to update you again with a more concrete date when the documentation is
    updated - just let us know.

So I don't think we'll have to wait too long.

**Comment by terinjokes** [comment]
Released in [v2.6.0](https://github.com/cloudflare/node-cloudflare/releases/tag/v2.6.0).
